packages/melody-test-utils/__tests__/__fixtures__/input.twig

```diff
@@ -0,0 +1,3 @@
+<div>
+  <input type="text" ref="{{ input }}" value="{{ value }}"/>
+</div>
```

Comment:
We're missing a newline here. :)

Reply:
This should be fixed in the latest commit!